modules/react/popup/README.md

```diff
@@ -285,150 +135,154 @@ stack, all popups will close rather than only the topmost one.
 If `forwardRef` is provided, it will be the same as `stackRef`. If `forwardRef` is not provided`,
 this hook will create one and return it.
 
-This hook should be used by all stacked UIs unless using the `Popper` component.
+This hook should be used by all popups unless using the `Popper` component.
 
 Example:
 
 ```tsx
-const [open, setOpen] = React.useState(false);
-const stackRef = usePopupStack();
-
-const closePopup = () => {
-  setOpen(false);
-};
+const model = usePopupModel();
+usePopupStack(model.state.stackRef, model.state.targetRef);
 
 // add some popup functionality
-useCloseOnOutsideClick(stackRef, closePopup);
-useCloseOnEscape(stackRef, closePopup);
-```
-
-### usePopup
-
-Convenience hook for common Popups used as non-modal dialogs. It provides props to mix into
-composite parts of the Popup pattern.
-
-```tsx
-import {
-  SecondaryButton,
-  DeleteButton,
-  Popper,
-  Popup,
-  usePopup,
-  useCloseOnOutsideClick,
-  useCloseOnEscape,
-} from '@workday/canvas-kit-react';
-
-const MyDeleteButton = ({onConfirm}) => {
-  const {targetProps, closePopup, popperProps, stackRef} = usePopup();
-
-  // popup traits
-  useCloseOnOutsideClick(stackRef, closePopup);
-  useCloseOnEscape(stackRef, closePopup);
-
-  return (
-    <>
-      <DeleteButton {...targetProps}>Delete Item</DeleteButton>
-      <Popper placement="bottom" {...popperProps}>
-        <Popup heading="Delete Item" handleClose={closePopup}>
-          <p>Are you sure you'd like to delete?</p>
-          <DeleteButton onClick={onConfirm}>Yes</DeleteButton>
-          <SecondaryButton onClick={closePopup}>No</SecondaryButton>
-        </Popup>
-      </Popper>
-    </>
-  );
-};
+useCloseOnOutsideClick(model);
+useCloseOnEscape(model);
+
+return (
+  <>
+    <button ref={model.state.targetRef}>Open Popup</button>
+    {model.state.visibility !== 'hidden'
+      ? ReactDOM.createPortal(<div>Popup Contents</div>, model.state.stackRef.current)
+      : null}
+  </>
+);
 ```
 
 ## useAssistiveHideSiblings
 
 ```ts
-useAssistiveHideSiblings(ref: React.RefObject<HTMLElement>): void
+useAssistiveHideSiblings(model: PopupModel): {}
```

Comment:
Should we just link to the storybook mdx/docs for this stuff so we don't have it duplicated? That's what we've done elsewhere

Reply:
I think so... But where to link? What's the Storybook URL? Do we just link to Github Pages? This would mean the link doesn't work until we merge to master.

Reply:
I did a link similar to Breadcrumbs - a link to the master storybook. I'd like to have a more robust solution later.